CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ## Master
 
+* Add an error message for :podspec pods not matching the version in Pods and on disk  
```

Comment:
@orta this is in the wrong section perhaps when the branch change occurred it got messed up.